inference-engine/src/transformations/src/transformations/convert_opset1_to_legacy/convert_matmul_to_fc_or_gemm.cpp

```diff
@@ -107,7 +107,7 @@ void ngraph::pass::ConvertMatMulToFCorGemm::convert_matmul() {
              std::dynamic_pointer_cast<opset1::FakeQuantize>(fc_input_b.get_node_shared_ptr())) &&
             std::count_if(shape_b.begin(), shape_b.end(), [](size_t x) {
                 return x != 1;
-            }) <= 2) {
+            }) <= 2 && !matmul->get_transpose_a()) {
```

Comment:
If it's plugin specific we need to handle it somehow via transformations callback

Reply:
For sure. That's why the PR is marked like a draft.
First of all I wanna understand performance impact on CPU, and after that to discuss possible transformation pipeline customization with you.